src/Application/SqlScripts/SqlTraits.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Ravenfire\Magpie\Application\SqlScripts;
+
+/**
+ * Traits used by the sql scripts
+ */
+trait SqlTraits
+{
+    /**
+     * Limits the width of table columns to 12 characters.
+     *
+     * @param $results
+     * @param $db_columns
+     * @return array
+     */
+    public function setStrLen($results, $db_columns)
```

Comment:
I like this function, but not this name (names are hard). Because its doing more than just setting a string length. It's also handling the results.

So, maybe handleResults? or parseResults? Or something like that?